test/sources/Basic.sol

```diff
@@ -32,4 +34,15 @@ contract Basic {
         emit BasicEvent(addr, val);
     }
 
+    function firesStringEvent(string memory _str) public {
+        emit StringEvent(_str);
+    }
+
+    function firesIndexedStringEvent(string memory _str) public {
+        emit IndexedStringEvent(_str);
+    }
+
+    function firesIllegalUtf8StringEvent() public {
+        emit StringEvent('�������');
```

Comment:
Try `emit StringEvent(string(0x9f90e274657374))`.

Reply:
@cag 

solc is not letting me do this (have tried a few variants). Keep getting something along these lines:
```
TypeError: Explicit type conversion not allowed from "int_const 44913823586808692" 
           to "string memory".
        emit StringEvent(string(0x9f90e274657374));
                         ^----------------------^
```

Reply:
My bad, I don't usually use bytes literals. Try `emit StringEvent(string(hex"9f90e274657374"))`

Reply:
@cag Ah that works but it's detecting that it's invalid. Also tried w/ 0.4.26 and it's the same...
```
TypeError: Explicit type conversion not allowed from "literal_string (contains invalid UTF-8 sequence 
           at position 0)" to "string memory".
        emit StringEvent(string(hex"9f90e274657374"));
                         ^-------------------------^
```

Reply:
We don't have to use this kind of test fwiw, we could also try to trigger it using a raw log, [as shown here][1]. 


[1]: https://github.com/ethereum/web3.js/blob/3e10668745ddbff8c785351f36754343ec680acf/test/event.decode.js#L49-L57

Reply:
Also tried using a raw log style test.